src/ape/contracts/base.py

```diff
@@ -21,7 +21,7 @@
     TransactionNotFoundError,
 )
 from ape.logging import logger
-from ape.types import AddressType, ContractLog, LogFilter
+from ape.types import AddressType, MockContractLog, BaseContractLog, ContractLog, LogFilter
```

Comment:
Then won't need this
```suggestion
from ape.types import AddressType, MockContractLog, ContractLog, LogFilter
```